src/test/java/uk/gov/mca/beacons/api/gateways/NoteGatewayImplTest.java

```diff
@@ -64,4 +79,20 @@ void shouldCreateANoteEntityFromANote() {
     assertThat(createdNote.getFullName(), is(fullName));
     assertThat(createdNote.getEmail(), is(email));
   }
+
+  @Test
+  void shouldSetCreatedDateIfNotProvided() {
+    final Note note = new Note();
+    final NoteEntity createdEntity = new NoteEntity();
+
+    when(noteRepository.save(noteCaptor.capture())).thenReturn(createdEntity);
+
+    noteGateway.create(note);
+
+    final NoteEntity entity = noteCaptor.getValue();
+    assertThat(
+      entity.getCreatedDate().getDayOfYear(),
+      is(equalTo(LocalDateTime.now().getDayOfYear()))
+    );
+  }
```

Comment:
We chatted about mocking system time :-) https://www.baeldung.com/java-override-system-time